src/components/Input/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { FC, useState, KeyboardEvent } from "react";
+
+import { SearchInputProps } from "./interface";
+
+import SuggestionsListComponent from "./suggestionsListComponent";
+import {
+  Form,
+  Input,
+  Label,
+  SearchIcon,
+  RemoveIcon,
+  FormWrapper,
+  Button,
+} from "./styledComponents";
+
+const SearchInput: FC<SearchInputProps> = ({ suggestions }) => {
```

Comment:
I'd rename the component and the file to something more descriptive of this functionality, like: `SearchAutocomplete`, `ListAutocomplete`, `Combobox`? WDYT?

Reply:
I could go with SearchAutocomplete or Combobox